cmd/http-server/startcmd/start.go

```diff
@@ -95,43 +156,65 @@ func createStartCmd(srv server) *cobra.Command {
 		Short: "Start http server",
 		Long:  "Start http server",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			hostURL, err := getUserSetVar(cmd, hostURLFlagName, hostURLEnvKey, false)
-			if err != nil {
-				return err
+			hostURL, hostURLErr := getUserSetVar(cmd, hostURLFlagName, hostURLEnvKey, false)
```

Comment:
it would be nice to use `err`

Reply:
I updated existing code to increase the patch coverage above 85% (since we have couple of files in the project).